### DISCUSSION TITLE
Custom hourly timer with a restart

### DISCUSSION THREAD

**Discussion by kayarre** [Miksus/rocketry#188] [Q&A]
Hi,
   I am trying to figure out how I can extend an minutely condition or another customized way to implement running a task every 9 minutes and 14 minutes, but at the end of the task return a "last modified" datetime object, and reset the timer to count from the returned datetime object. 
 
 e.g. At 9 minutes run the task which returns a new timestamp that was 3 minutes in the past. I want to update the timer clock to count the 9 minutes starting from the 3 minutes in the past, and would mean the task gets run again in 6 minutes from the present.
 
Basically a way to offset a timer based on new information.
